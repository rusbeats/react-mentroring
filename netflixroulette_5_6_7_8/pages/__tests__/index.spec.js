import React from 'react';
import { shallow } from 'enzyme';
import { Index } from '../index';

describe('Index', () => {
    it('should render correctly (snapshot)', () => {
        const component = shallow(<Index />);
        expect(component).toMatchSnapshot();
    });
});
