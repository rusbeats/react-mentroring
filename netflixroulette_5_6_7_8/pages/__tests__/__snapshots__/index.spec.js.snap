// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render correctly (snapshot) 1`] = `<HomePage />`;
